github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -1,5 +1,56 @@
 postsubmits:
   kubevirt/kubevirtci:
+    - name: publish-kubevirtci-s390x
+      branches:
+      - main
+      always_run: true
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      max_concurrency: 1
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-gcs-credentials: "true"
+        preset-github-credentials: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: s390x-prow-virtual
+      spec:
+        volumes:
+        - hostPath:
+            path: /dev
+            type: Directory
+          name: devices
+        containers:
+        - image: quay.io/kubevirtci/golang:v20240814-9b6c4af
+          command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/bash"
+          - "-c"
+          - >
+            cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
+            ./publish.sh &&
+            SHORT_SHA=$(git rev-parse --short HEAD) &&
+            echo "$(git tag --points-at HEAD | head -1)" > s390x-$SHORT_SHA &&
+            gsutil cp ./s390x-$SHORT_SHA gs://kubevirt-prow/release/kubevirt/kubevirtci/s390x-$SHORT_SHA
+          # docker-in-docker needs privileged mode
+          env:
+          - name: GIMME_GO_VERSION
+            value: "1.19.9"
```

Comment:
The go version will have to be updated to match the x86 job

Reply:
done